webview-ui/src/components/AnalysisPage/AnalysisPage.tsx

```diff
@@ -344,13 +346,24 @@ const AnalysisPage: React.FC = () => {
                 </StackItem>
               </Stack>
             </PageSection>
-            {isWaitingForSolution && (
+            {(isWaitingForSolution || isWaitingForUserInteraction) && (
               <Backdrop>
                 <div style={{ textAlign: "center", paddingTop: "15rem" }}>
                   <Spinner size="lg" />
                   <Title headingLevel="h2" size="lg">
-                    Waiting for solution confirmation...
+                    {isWaitingForUserInteraction 
+                      ? "Waiting for user action..."
+                      : "Waiting for solution confirmation..."}
                   </Title>
+                  {isWaitingForUserInteraction && (
+                    <Button
+                      variant="primary"
+                      onClick={() => dispatch(openResolutionPanel())}
+                      style={{ marginTop: "1rem" }}
+                    >
+                      Open Resolution Panel
+                    </Button>
+                  )}
```

Comment:
I think you should just always show the button.

```suggestion
                    <Button
                      variant="primary"
                      onClick={() => dispatch(openResolutionPanel())}
                      style={{ marginTop: "1rem" }}
                    >
                      Open Resolution Panel
                    </Button>
```